src/core/GAT.jl

```diff
@@ -647,12 +647,12 @@ function instance_code(mod, instance_types, instance_fns, external_fns)
   bound_fns = OrderedDict(parse_function_sig(f) => f for f in bound_fns)
   instance_fns = Dict(parse_function_sig(f) => f for f in instance_fns)
   for (theory, f) in bound_fns
```

Comment:
I didn't catch this the first time, but `theory` here should have been kept as `sig` because the object is the signature of a Julia function not the signature of a GAT.

Reply:
Fixed